content/en/dashboards/guide/custom_time_frames.md

```diff
@@ -38,30 +38,40 @@ Both fixed and relative custom time frames are supported:
 
-| Format       | Examples                                                                         | Notes                                                     |
-|--------------|----------------------------------------------------------------------------------|-----------------------------------------------------------|
-| `N{unit}`    | 3m<br>3 min<br>3h<br>3 hours<br>3d<br>3 days<br>3w<br>3 weeks<br>3mo<br>3 months | Displays the past N units, for example: the past 3 months |
-| `today`      |                                                                                  | Displays the current calendar day until present           |
-| `yesterday`  |                                                                                  | Displays the full previous calendar day                   |
-| `this month` |                                                                                  | Displays the current calendar month until present         |
-| `last month` |                                                                                  | Displays the full previous calendar month                 |
-| `this year`  |                                                                                  | Displays the current calendar year until present          |
-| `last year`  |                                                                                  | Displays the full previous calendar year                  |
-
-* The following strings are accepted for any `{unit}` in a relative date:
+| Format         | Examples                                                                         | Notes                                                     |
+|----------------|----------------------------------------------------------------------------------|-----------------------------------------------------------|
+| `N{unit}`      | 3m<br>3 min<br>3h<br>3 hours<br>3d<br>3 days<br>3w<br>3 weeks<br>3mo<br>3 months | Displays the past N units, for example: the past 3 months |
+| `today`        | Nov 9, 12:00 am - Nov 9, 3:15 pm                                                 | Displays the current calendar day until present           |
+| `yesterday`    | Nov 8, 12:00 am - Nov 9, 12:00 am                                                | Displays the full previous calendar day                   |
+| `this month`   |                                                                                  | Displays the current calendar month until present         |
+| `last month`   |                                                                                  | Displays the full previous calendar month                 |
+| `this year`    |                                                                                  | Displays the current calendar year until present          |
+| `last year`    |                                                                                  | Displays the full previous calendar year                  |
```

Comment:
Is it expected not to have any examples for this month, last month, this year, last year?

Reply:
There weren't any listed, but we can add them if you think it would be helpful!

Reply:
I think it would make sense, it felt weird reading the table where some fields are blank